sklearn/sklearn-template/template/scripts/predict.py

```diff
@@ -0,0 +1,62 @@
+# Copyright 2019 Google Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ==============================================================================
+
+"""Helper function for requesting an online prediction."""
+
+import googleapiclient.discovery
+
+
+def predict_json(project, model, datas, version=None):
```

Comment:
Actually my intention to have it is simply to demonstrate how to request prediction through python client, since i believe it would be more common used compared the bash script for the use case of online prediction.

At first i thought putting a util function in utils, which people can refer to. But it would be wired to include google api client dependency in the trainer package. I don't find a better place to put it, therefore I made a python scripts under scripts/.

There I do think there is no need to make it consistent with predict.sh. And it should be pretty easy for users to change it themselves.

Reply:
Ok. Yeah it's not exactly part of model development. I think `scripts` would be fine for now.
If you'd like to keep the script without command-line args, would suggest getting the value for `project_id` in `__main__` using the `PROJECT_ID` environmental variable so it's consistent with the other scripts. You can use `os.environ`.

Reply:
just pushed.

Reply:
got it, thanks